app/views/record.html

```diff
@@ -49,7 +66,8 @@
 
         <div class="record-actions__links">
           {% if record.status | getCanDefer %}
-            <p class="govuk-body govuk-!-margin-bottom-0"><a href="{{referrer}}/defer" class="govuk-link--no-visited-state">Defer</a> or <a href="{{referrer}}/withdraw" class="govuk-link--no-visited-state">withdraw</a> this trainee</p>
+            <p class="govuk-body govuk-!-margin-bottom-0">
+            <a href="{{referrer}}/defer" class="govuk-link--no-visited-state">Defer</a> or <a href="{{referrer}}/withdraw" class="govuk-link--no-visited-state">withdraw</a> this trainee</p>
```

Comment:
```suggestion
              <a href="{{referrer}}/defer" class="govuk-link--no-visited-state">Defer</a> or <a href="{{referrer}}/withdraw" class="govuk-link--no-visited-state">withdraw</a> this trainee
            </p>
```

Trying to fix indentation here. Not sure github has got it right though.